fairlearn/metrics/__init__.py

```diff
@@ -46,7 +46,7 @@
     selection_rate,
     _mean_overprediction,
     _mean_underprediction,
-    )
+    count)
```

Comment:
I think it would make sense for it to live in the `_extra_metrics.py` file, but in your PR it seems to be in its own file. Can you move it into the file? I'm not entirely sure how it's actually working right now since there should be no `count` in `_extra_metrics`, so what is it actually importing?

Reply:
> I think it would make sense for it to live in the `_extra_metrics.py` file, but in your PR it seems to be in its own file. Can you move it into the file? I'm not entirely sure how it's actually working right now since there should be no `count` in `_extra_metrics`, so what is it actually importing?

I was doing something similar to `fairlearn.metrics._selectrion_rate`. The selection rate function is in its own file, but it's imported through `fairlearn.metrics._extra_metrics` here: https://github.com/fairlearn/fairlearn/blob/main/fairlearn/metrics/_extra_metrics.py#L14
Maybe I misunderstood this comment and its answer: https://github.com/fairlearn/fairlearn/issues/737#issuecomment-821566124

Reply:
True, but you're not importing `count` in `_extra_metrics.py`. Anyway, it's simpler to have it in one file 😄